metabase/metabase
# Command palette/main

Closes https://github.com/metabase/metabase/issues/39479

### Description
This PR handles Milestones 1 & 2 of https://github.com/metabase/metabase/issues/39259

### How to verify
Load the homepage and press ctrl/cmd + k. This will open the command palette. From here you should see recent items, be able to create new entities, and more. Functionality is currently largely tied to navigation. Opening the command palette in the admin app should also allow you to search for specific settings.

### Demo

_Upload a demo video or before/after screenshots if sensible or remove the section_

### Checklist
